ansible/roles/kind/kind_cluster_create/vars/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+project: "{{ lookup('env', 'PROJECT') }}"
```

Comment:
since the `project` variable is used across playbooks to access the cluster name, maybe use group vars instead of defining it per role?

Reply:
added